packages/cli-kit/src/private/node/ui/components/Banner.test.tsx

```diff
@@ -0,0 +1,58 @@
+import {Banner} from './Banner.js'
+import {renderString} from '../../ui.js'
+import {describe, expect, test} from 'vitest'
+import React from 'react'
+
+describe('Banner', async () => {
+  test('renders with a border for success', async () => {
+    const {output} = renderString(<Banner type="success" />)
+
+    expect(output).toMatchInlineSnapshot(`
+      "[32m╭─[39m success [32m────────────────────────────────────────────────────────────────────╮[39m
```

Comment:
The only thing to test here are text colors so I kept the ansi escape sequences.